app/src/androidTest/java/com/github/polybooks/database/SaleDatabaseTest.kt

```diff
@@ -245,4 +246,36 @@ class SaleDatabaseTest {
         future = db.querySales().getN(1, 0)
         assertTrue(future.get().size <= 1)
     }
+
+    @Test
+    fun addDelete(){
+        val saleTest = Sale("test-tqwjdhsfalkfdhjasdhlfkahdfjklhdjhfl.adfjasdhflka-adjklshfjklasdhfjklhasd",
+            301943, 666f,
+            BookCondition.WORN,
+            Timestamp(com.github.polybooks.database.format.parse("2016-05-05")!!),
+            SaleState.RETRACTED )
```

Comment:
Couldn't that be a class private val instead of inside the test? Since it's duplicated in fun Delete()?